src/network/MissionControlProtocol.cpp

```diff
@@ -32,6 +35,16 @@ const std::chrono::milliseconds HEARTBEAT_TIMEOUT_PERIOD = 3000ms;
  */
 static void stopAllJoints();
 
+/**
+ * Get the steer rotations of all wheels as a swerve_rots_t.
+ */
+static control::swerve_rots_t getAllSteerRotations();
+
+/**
+ * Set the steer power of all wheels with a swerve_commands_t.
+ */
+static void setAllSteerPowers(control::swerve_commands_t commands);
```

Comment:
Makes sense, what if we just changed the methods of the power repeat task to also execute it now? That way we can just call `setTurnInPlaceCmdVel` or whatever and it will both schedule the power repeat *and* send the command now. That way the utility functions can only exist in MCPTasks